packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -829,6 +829,11 @@ class SizedBox extends SingleChildRenderObjectWidget {
       height = double.INFINITY,
       super(key: key, child: child);
 
+  SizedBox.fromSize({ Key key, Widget child, Size size })
+    : width = size?.width,
```

Comment:
I was unable to figure out a way to make it one. Even if you assume size is non-null, initializing the fields to size.width and size.height produces a 'not a const expression' error.